tests/test_EVMAssembler.py

```diff
@@ -44,6 +44,28 @@ def test_JUMPI(self):
         self.assertTrue(insn.mnemonic == 'JUMPI')
         self.assertTrue(insn.is_branch)
 
+    def test_pre_byzantium(self):
+        insn = EVMAsm.disassemble_one(b'\x57', fork='pre-byzantium')
+        self.assertTrue(insn.mnemonic == 'JUMPI')
+        self.assertTrue(insn.is_branch)
+        insn = EVMAsm.disassemble_one(b'\xfd', fork='pre-byzantium')
+        self.assertTrue(insn.mnemonic == 'INVALID')  # REVERT added in byzantium
+
+    def test_byzantium_fork(self):
+        insn = EVMAsm.disassemble_one(b'\x57', fork='byzantium')
+        self.assertTrue(insn.mnemonic == 'JUMPI')
+        self.assertTrue(insn.is_branch)
+        insn = EVMAsm.disassemble_one(b'\x1d', fork='byzantium')
+        self.assertTrue(insn.mnemonic == 'INVALID')
+        insn = EVMAsm.disassemble_one(b'\xf5', fork='byzantium')
+        self.assertTrue(insn.mnemonic == 'INVALID')
+
+
+    def test_constantinople_fork(self):
+        insn = EVMAsm.disassemble_one(b'\x1d', fork='constantinople')
+        self.assertTrue(insn.mnemonic == 'SAR')
+        self.assertTrue(insn.is_arithmetic)
```

Comment:
Can we also test all the others instructions that have been added?

Reply:
Yes. I thought about it as well. I will test all of them.

Reply:
All new instructions are tested now.